db/note.php

```diff
@@ -67,6 +94,28 @@ public static function fromFile(File $file, Folder $notesFolder, $tags=[]){
         $note->resetUpdatedFields();
         return $note;
     }
+    /**
+     * @param File $file
+     * @return static
+     */
+    public static function fromException($message,File $file,Folder $notesFolder,$tags=[]){
+        $note = new static();
+        $note->setId($file->getId());
+        $note->setErrorMessage($message);
+        $note->setError(true);
+        $note->setContent($message);
+        $note->setModified(null);
+        $note->setTitle($message); // remove extension
```

Comment:
The title should be the same like in the non-error case (note's title from filename). Or is the filename encrypted, too?

Reply:
It currently shows the exception / error along with the filename in both the title and content fields ie "Decryption Error: (somefile.txt) exception blahblah". I guess this function is also fulfilled by the error popups so it seems a bit redundant, the user can just click to see the error.